docker-compose.yml

```diff
@@ -71,8 +71,8 @@ x-airflow-common: &airflow-common
     dockerfile: Dockerfile.local
   environment: &airflow-common-env
     AIRFLOW__CORE__EXECUTOR: CeleryExecutor
-    AIRFLOW__DATABASE__SQL_ALCHEMY_CONN: postgresql+psycopg2://inspirehep:inspirehep@db/inspirehep
-    AIRFLOW__CELERY__RESULT_BACKEND: db+postgresql://inspirehep:inspirehep@db/inspirehep
+    AIRFLOW__DATABASE__SQL_ALCHEMY_CONN: postgresql+psycopg2://inspirehep:inspirehep@db/workflows
```

Comment:
don't we need to create `workflows` db as well in postgres?

Reply:
Hello Harris this db is now being created in the `init.sql` file. We can talk about it once your back.
Please enjoy your vacations :)